### DISCUSSION TITLE
Maximum radiative fraction

### DISCUSSION THREAD

**Discussion by jonmaddock** [ukaea/PROCESS#2886] [Ideas]
In GitLab by @mkovari on Jun 2, 2023, 09:50

We limit the radiation fraction f<sub>rad</sub> using constraint 17.  For example, the input below sets the maximum radiation fraction to 90%.  This works and gives the output below, but it requires the user to set the upper bound of the f-value to the maximum radiation fraction.  It would be more intuitive to introduce a new input variable: `frad_max = Maximum radiation fraction`.  The upper bound for the f-value `fradpwr` would then be 1 as is usual for f-values.  
.  
.

```
icc = 17 
* DESCRIPTION:   Radiation Fraction Upper Limit

ixc = 28
fradpwr = 0.86364
boundu(28) = 0.9
```
```
                   fradpwr       =  9.0000E-01 is at or above its upper bound:  9.0000E-01
...
 Total power deposited in plasma (MW)                                     (tot_power_plasma)        5.796E+02  OP 
...
 Total radiation power from inside LCFS (MW)                              (pradmw)                  5.216E+02  OP 
 LCFS radiation fraction = total radiation in LCFS / total power deposite (rad_fraction_LCFS)       9.000E-01  OP 
```

**Comment by jmorris-uk** [comment]
Agree this should be changed to act like a normal constraint equation, I see now reason why it should use the f-value as the limit. 

It's quite a simple fix. @ajpearcey can you raise an issue and assign someone to implement.
